vignettes.Rmd

```diff
@@ -183,7 +183,7 @@ Here are some resources we've found helpful:
 
 -   Literally anything written by Kathy Sierra.
     She is not actively writing at the moment, but her content is mostly timeless and is full of advice about programming, teaching, and how to create valuable tools.
-    See her original blog, [Creating passionate users](https://headrush.typepad.com/), or the site that came after, [Serious Pony](https://seriouspony.com/blog/).
```

Comment:
What do you think about using the Wayback Machine instead?

https://web.archive.org/web/20230411010913/https://seriouspony.com/blog/

Reply:
Nice idea. I've added another commit with the archived link in case you want to go with that.